src/components/project-card/project-card.test.tsx

```diff
@@ -0,0 +1,65 @@
+import Project from '@/components/project-card/project-card';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+describe('Project', () => {
+  const mockProjectData = {
+    title: 'Test Project',
+    description: 'Test Description',
+    links: [
+      {
+        title: 'GitHub',
+        url: 'https://github.com/example',
+      },
+    ],
+    languages: ['JavaScript', 'TypeScript'],
+    tools: ['React', 'Next.js'],
+    technologies: ['MUI', 'Jest'],
+    programAreas: ['Web Development'],
```

Comment:
`programAreas` is for the project purpose. Like here: https://www.hackforla.org/projects/